### DISCUSSION TITLE
Thanks for the awesome tool

### DISCUSSION THREAD

**Discussion by CodeJawn** [vzhd1701/enex2notion#49] [General]
I just wanted to take a moment to express my appreciation.  I had issues with the official notion importer (missing images).  I came across this tool and it worked flawlessly for me.  I also appreciated the robust documentation for installing and using it.  Thanks for taking the time to make and publish this!

**Comment by vzhd1701** [comment]
I am glad to hear you found it helpful!

**Comment by EwanMcP** [comment]
Came to say the same - thank you. Many years of Evernote content which couldn't be reliably or properly imported, and I was close to giving up for the time being when I found your tool. Thank you!

**Comment by vcavett** [comment]
Another thank you for building and maintaining this tool. This has been a lifesaver as we've started migrating years of notes out of EN.

**Comment by jasonab** [comment]
The native importer choked on so many of my notes, this tool was indispensable!

**Reply by elaquiambao** [reply]
Big help! Thank you so much!
